### DISCUSSION TITLE
Use svg path as bounds

### DISCUSSION THREAD

**Discussion by mutheusalmeida** [pmndrs/use-gesture#533] [General] [answered]
Hi, I'm trying to make something similar to what the native animateMotion API from SVG does but I wanted to use mouse and touch gestures for that. Is it possible to use svg coordinates as the bound of use-gesture?

**Comment by dbismut** [comment, marked as answer]
No. You can pass a node to use gesture bounds option and it will use `getBoundingClientRect` but this is obviously different than what you need.
